mod/kalvidassign/lang/en/kalvidassign.php

```diff
@@ -141,3 +141,4 @@
 $string['privacy:metadata:kalvidassignquickgrade'] = 'Quick grading preference for assignment submissions.';
 $string['privacy:markedsubmissionspath'] = 'markedsubmissions';
 $string['privacy:submissionpath'] = 'submission';
+$string['search:activity'] = 'Kaltura Video Assign';
```

Comment:
Hi @yedidiaklein, 
Thank you for your contribution.
Can you please change the text here to 'Kaltura Media Assignment'?
thanks